### PR TITLE
Allow disabling of Dashboard Live Polling configuration

### PR DESCRIPTION
**Pull request by erick-tmr**
This PR adds a simple configuration to disable dashboard live polling depending on a configuration.

# Considerations
I didn’t make more efforts to hard enforce this rule, like checking this config in JS code or something like this, because Good Job is supposed to be an dev only environment and people should know the consequences of editing HTML and trying to skip some configuration "brute forcibly".
But I'm open to other suggestions.

Issue https://github.com/bensheldon/good_job/issues/1234

### CODE REVIEW

File: app/views/good_job/shared/_navbar.erb [hunk 1]
```diff
@@ -60,7 +60,7 @@
 
         <li class="nav-item d-flex flex-column justify-content-center">
           <div class="form-check form-switch m-0">
-            <%= check_box_tag "live_poll", params.fetch("poll", 30), params[:poll].present?, role: "switch", class: "form-check-input" %>
+            <%= check_box_tag "live_poll", params.fetch("poll", 30), params[:poll].present?, role: "switch", class: "form-check-input", disabled: !GoodJob.configuration.dashboard_live_poll_enabled %>
```
Comment (bensheldon) [inline app/views/good_job/shared/_navbar.erb:63]: I think we also want to make sure that it is always unchecked when disabled to:

```suggestion
            <%= check_box_tag "live_poll", params.fetch("poll", 30), (GoodJob.configuration.dashboard_live_poll_enabled && params[:poll].present?), role: "switch", class: "form-check-input", disabled: !GoodJob.configuration.dashboard_live_poll_enabled %>
```

**Review by bensheldon — COMMENTED** [review]
This looks good! Would you mind adding a simple system spec for this? 

(I'm also happy to make that change too if you find it difficult)

File: app/views/good_job/shared/_navbar.erb (hunk 1 above)
Comment (erick-tmr) [reply app/views/good_job/shared/_navbar.erb:63]: nice, this looks good

**Comment by erick-tmr** [comment]
> This looks good! Would you mind adding a simple system spec for this?
> 
> (I'm also happy to make that change too if you find it difficult)

Sure, I'll try to add those

**Comment by erick-tmr** [comment]
hey @bensheldon just added the specs, can you take a look again?

**Comment by erick-tmr** [comment]
Looks like some tests failed because of connection timeouts, I can’t retry those by the way :/

**Comment by bensheldon** [comment]
No sweat! This is a new flake, unrelated to your changes.

**Pull request merged**